### DISCUSSION TITLE
Medusa Admin UI products page needs bulk editing

### DISCUSSION THREAD

**Discussion by Kainkainkain** [medusajs/medusa#5889] [Feature Requests]
Currently, making big changes to your shops products is best done outside of medusa in excel, and then importing. 
This is because there are no easy management controls for editing a lot of products at once. 

Example scenarios that currently are very difficult to do in Medusa:
"I'm going to change my Collections from Bottoms to Pants, Shorts, Socks"
"I've already set up the store, but now we have stock numbers I can finally update them, 100 x of each"
"I'm using tags in my store front for refined filtering and in query params to make special filtered page views, but now I want to apply a series of different tags to selected groups of products"
"I want to delete 200 products"
"60 of these items have the same weight 1200g, which effects shipping companies prices so its important. I want to update them all in one go"

Medusa Admin UI products page needs bulk editing:

- bulk **product deletion**, 
- bulk **adding tags**, 
- bulk deleting tags
- bulk assign type
- bulk assign collection (if exists, overwrite)
- bulk metadata entry (if clash, overwrite)
- bulk edit discountable true or false
- bulk edit subtitle
- bulk edit country of origin
- bulk edit width, height, length, weight

More tricky, but definitely it would be great if we could bulk edit variants, maybe with an option:
'Edit first variant', 'edit if only 1 variant', 'edit all variants'
- bulk edit width, height, length, weight
- bulk update **Quantity in stock**
- bulk update Manage inventory to true or false
- bulk update Allow backorders to true or false
- bulk edit country of origin
- bulk edit metadata (overwrite if clash)

## UI concept: copy wordpress maybe
Just have checkboxes on the left of each product, and at the top a drop down or a expanding component of 'bulk edit' controls to apply to the selected items.

**Comment by Deroswent** [comment]
Dude, Medusa is not a CMS. Medusa is a data provider! Like a mysql database for a regular website, but via API, that's what medusa is.

And what you want is CMS functions!  These are far from priority features for medusa.
Just integrate Medusa with some headless CMS like Strapi and you'll get all these features and more. It's in the official documentation.
